### ISSUE TITLE
Manage who can view reports

### ISSUE THREAD

**Issue by viettran97118** [getredash/redash#6612]
Manage who can view reports, add more permission editor, view, comment ?

![image](https://github.com/getredash/redash/assets/148329464/3eda1de2-7d28-4beb-9487-d35111a6ef96)

**Comment by guidopetri** [comment]
I don't think we have fine-grained functionality like that, but it would be a good addition. Thanks for the suggestion!

**Comment by viettran97118** [comment]
Yes, i know Redash manage data resouce, is not manage dashboard or query. So, it is convenience for reporting manager.
